book.toml

```diff
@@ -61,3 +61,7 @@ smart-punctuation = true
 "platforms/zulip/triagebot.html" = "/triagebot/index.html"
 "core/blogs.html" = "/platforms/blogs.html"
 "core/index.html" = "/governance/council.html"
+
+[output.linkcheck]
+follow-web-links = false
+optional = true
```

Comment:
Question: does this block? Note that in rustc-dev-guide we [resorted to disabling linkcheck on PR CI](https://github.com/rust-lang/rustc-dev-guide/pull/2296) because checking external weblinks (even github ones) can cause spurious PR CI failures (random permission failures, anti-crawler technologies, etc.), and was unfortunately preventing actual content updates from getting through (i.e. it was more noisy than helpful unfortunately).